app/views/analytics/index.html.erb

```diff
@@ -147,26 +147,59 @@
     </div>
   </div><!-- row   -->
 
-  <!-- Comments List-->
   <div class="row">
-    <div class="col-sm-12">
-      <div class="chart-wrapper">
-        <div class="chart-title background-orange">
-          Recent Comments
-        </div>
-        <ul class="scroller">
-          <% @most_recent_comments.each do |comment| %>
-            <li>
-              <blockquote class="blockquote">
-                <p><%= link_to_case_title(comment.commentable, 25) %> -- <%= comment_created_at(comment) %></p>
-                <p><%= link_to_comment(comment, 100) %></p>
-                <small><%= link_to_comment_user(comment) %></small>
-              </blockquote>
-            </li>
-          <% end %>
-        </ul>
+    <!-- Editors List-->
+      <div class="col-sm-4">
+        <div class="chart-wrapper">
+          <div class="chart-title background-orange">
+            Most Recent Editors - Last 30 days <%= image_tag('help_icon.png', data: {:toggle => 'popover', :container => 'body', :trigger => 'hover',:placement => "bottom", :title => "Active Editors", :content => "This metric shows the number of cases added or updated by editors in the past 30 days."})%>
+          </div>
+          <div class="chart-stage">
+           <table class='table leader-list'>
+             <tr>
+              <th>Editor</th>
+              <th>Updates</th>
+             </tr>
+
+             <% case_updaters_count(10).each do |who| %>
+               <% user_id = who[0].to_i %>
+               <% user_object = who[1] %>
+               <tr>
+                 <% unless user_id == 0 %>
+                   <td><%= @user = User.find(user_id).name %></td>
```

Comment:
Database queries need to go in the controller, not the views.  Plus, if you already have the user object, there shouldn't be a need to run a second query just to get their name - just pull it from the object.

Reply:
Not sure how to do this. `case_updaters_count` doesn't return the user object, only the user ids corresponding to the whodunnits in order of the most prolific editors. 

To get around this, I think I can add a a model method that counts the number of edits by a user over the past XX days. Then in the controller, create the instance variable that calls the method and returns the top ten editors.

Does that make sense?

Reply:
@campgurus I'd make it a model scope rather than a method, but that's a step in the right direction.